recipes/default.rb

```diff
@@ -37,6 +37,10 @@
   system  true
 end
 
+user_ulimit node["kafka"]["user"] do
+  filehandle_limit 'unlimited'
```

Comment:
IIRC "unlimited" doesn't work properly for open file descriptor limits, I've seen various docs recommend "100000" or "65536".